package.json

```diff
@@ -5,12 +5,18 @@
   "module": "dist/webexComponents.esm.js",
   "scripts": {
     "build": "rollup -c",
+    "blah": "build-storybook -o ./docs/storybook",
```

Comment:
Is this something that should be committed?

Reply:
nope, this is what he used for pair programming with me

Reply:
Let's get this removed then

Reply:
![](https://media0.giphy.com/media/2vs62IAzsdhFR76ZjB/giphy.gif #gifbar)